### DISCUSSION TITLE
how to hide story in sidebar

### DISCUSSION THREAD

**Discussion by williammyuan** [storybookjs/storybook#24849] [Help] [answered]
Hi, I am a new to use storybook, I only want to show Docs, how can I do?
<img width="1155" alt="企业微信截图_82654d04-fb13-475e-8a7d-23e9404d3841" src="https://github.com/storybookjs/storybook/assets/45476221/6cb871d4-3ccd-4ac3-b664-f5dde46c228a">

**Comment by joevaugh4n** [comment]
Hey there, and welcome to the world of Storybook!

If you want to only show docs, you can hide the sidebar by using Storybook's keyboard shortcuts 👇

![image](https://github.com/storybookjs/storybook/assets/122215197/81fc395e-bd4f-4c91-96bd-0d285df04b7a)

Does this help?

**Reply by williammyuan** [reply]
Thank you very much for your answer! I may not have expressed it clearly, what I mean is how to configure storybook, when using the storybook build command to generate a static site, the user only sees Docs after deployment.
Here is an example website:https://pigment.customink.com/?path=/docs/components-link--api

**Reply by jonniebigodes** [reply, marked as answer]
@williammyuan, thanks for reaching out to us with your question and providing us with some context. We appreciate it 🙏 . Storybook has a flag that can be added to your build command that allows you to build your documentation only and leave out the stories. Which you can then deploy to any hosting provider. You can either run the command as follows:
```shell
storybook build --docs
```
Or have it in a script in your `package.json` to help you streamline your workflow. For instance:
```json
"scripts": {
  "build-storybook-docs": "storybook build --docs"
}
```

Running one of the options will generate something similar to the following (for context, I'm using the example provided in our Design Systems for Developers tutorial [here](https://storybook.js.org/tutorials/design-systems-for-developers))

![storybook-build-docs-mode](https://github.com/storybookjs/storybook/assets/22988955/2bbeca4a-8e68-4d07-ad6e-7a900bb14e17)

As you can see by the image above, you only have the documentation and not the stories in the Storybook sidebar. If you want to read more about it, we have some documentation [here](https://storybook.js.org/docs/react/writing-docs/build-documentation) that can help you with your use case.

Let us know if we're able to help you out so that we can close out this discussion or continue to work with you on it so that you're able to solve your issue. 

Looking forward to hearing from you.

Have a fantastic day.

Stay safe

**Reply by joevaugh4n** [reply]
Thank you, @jonniebigodes!

**Reply by jonniebigodes** [reply]
No need to thank whatsoever @joevaugh4n, glad to help 🙂

**Reply by williammyuan** [reply]
@jonniebigodes This works for me, thank you.

**Reply by joevaugh4n** [reply]
Glad you were able to resolve it, @williammyuan!
